WcaOnRails/app/views/competitions/_competition_form.html.erb

```diff
@@ -206,22 +206,25 @@
         <%= f.input :base_entry_fee_lowest_denomination, as: :money_amount, currency: @competition.currency_code, disabled: disable_money_input, currency_selector: "#competition_currency_code" %>
 
         <script>
-          var bandFeeUSD = 0;
           var baseFeeUSD = -1;
-          var compLimit = null;
-          var tempLimit = null;
-
-          <% countryBand = CountryBand.where(iso2: @competition.serializable_hash[:country_iso2]).map(&:number)[0] %>
-          <% if !countryBand.nil? && countryBand > 0 %>
-            bandFeeUSD = <%= CountryBand::BANDS[countryBand][:value] %>;
-          <% end %>
 
+          // Note: unfortunately we cannot trigger a change on the currency
+          // code update, because we don't have access to the currency
+          // "subunit_to_unit" from the js world.
```

Comment:
I placed this comment because I wondered why we were not updating dues estimate on currency change, and then later understood we would need the `subunit_to_unit` for every currencies.